cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -30,6 +30,12 @@ import (
 const (
 	PoolOperator           = "zpool"
 	StatusNoPoolsAvailable = "no pools available"
+	ZpoolStatusDegraded    = "DEGRADED"
```

Comment:
This statuses might have been better off with some custom status type. However, good for now.